### ISSUE TITLE
BREW /test HTCPCP/1.0\r\n doesn't work

### ISSUE THREAD

**Issue by lapsio** [stephen/node-htcpcp#2]
HTCPCP requests doesn't work. HTCPCP shouldn't be HTTP based, it's just bad implementation. Nodejs http server rejects all non-http requests including htcpcp ones. htcpcp server must be raw tcp/ip based.

**Comment by stephen** [comment]
You're right. Recently worked on another node package that worked on the socket-level to parse similar-to-HTTP protocols ([httplike](https://github.com/stephencwan/httplike)) such as htcpcp. 

Will likely re-write this package to use httplike at some point.
